fabtests/pytest/common.py

```diff
@@ -27,6 +27,7 @@ def is_ssh_connection_error(exception):
 
 def has_ssh_connection_err_msg(output):
     err_msgs = ["Connection closed by remote host",
+                r"Connection closed by .* port 22",
```

Comment:
I am guessing we can just use `Connection closed`

Reply:
Thanks. Fixed.